pep-0570.rst

```diff
@@ -113,30 +153,44 @@ arguments by name::
 via the ``/`` marker, but this at the moment is only documentational,
 Python developers cannot write such syntax.
 
-In addition, there are some functions with particularly
+Besides, there are some functions with particularly
 interesting semantics:
 
 * ``range()``, which accepts an optional parameter
   to the *left* of its required parameter. [#RANGE]_
 
-* ``dict()``, whose mapping/iterator parameter is optional and
-  semantically must be positional-only.  Any externally
-  visible name for this parameter would occlude
-  that name going into the ``**kwarg`` keyword variadic
+* ``dict()``, whose mapping/iterator parameter is optional and semantically
+  must be positional-only.  Any externally visible name for this parameter
+  would occlude that name going into the ``**kwarg`` keyword variadic
   parameter dict! [#DICT]_
 
-Obviously one can simulate any of these in pure Python code
+One can simulate any of these in pure Python code
 by accepting ``(*args, **kwargs)`` and parsing the arguments
-by hand.  But this results in a disconnect between the
-Python function signature and what the function actually accepts,
+by hand.  However, this results in a disconnect between the
+Python function signature and what the function accepts,
 not to mention the work of implementing said argument parsing
-and the lack of clarity in the resulting signature.
+also, the lack of clarity in the resulting signature.
+
+As mentioned before, this syntax is already being used outside the
+CPython code base for similar use cases [#numpy-ufuncs]_ [#scipy-gammaln]_,
+remarking that these scenarios are not exclusive to CPython and the
+standard library.
+
+The main reason this notation is received with surprise currently
+by users is because it has not been adequately documented until very
+recently [#document-positional-only]_ and is not possible to use it
+in Python code. For these reasons, this notation is currently an
+oddity that appears only in CPython's APIs developed in C. Documenting
+the notation and making it possible to be used in Python code will
+certainly, eliminate this problem.
```

Comment:
```suggestion
certainly eliminate this problem.
```

I think "certainly" is still a bit strong here, I'd probably go with "likely" or something softer, but I don't feel strongly about it.